runtime_config_linux.go

```diff
@@ -39,8 +39,10 @@ type LinuxRuntime struct {
 	SelinuxProcessLabel string `json:"selinuxProcessLabel"`
 	// Seccomp specifies the seccomp security settings for the container.
 	Seccomp Seccomp `json:"seccomp"`
-	// RootfsPropagation is the rootfs mount propagation mode for the container
+	// RootfsPropagation is the rootfs mount propagation mode for the container.
 	RootfsPropagation string `json:"rootfsPropagation,omitempty"`
+	// disableNewPrivileges controls whether additional privileges could be gained by processes in the container.
```

Comment:
not sure if `golint` will ever be run, but I think it will want the comment to match the name (capital "**D**" for *DisableNewPrivileges*)